src/openrct2/paint/tile_element/Paint.Path.cpp

```diff
@@ -690,6 +690,8 @@ static void sub_6A3F61(
 
     if (dpi->zoom_level <= 1)
     {
+        bool bPaintScenery = true;
```

Comment:
That seems like a bit of an odd variable name - is that Hungarian notation? `paintScenery` should be clear enough, especially in this day of IDEs and actually useful compiler messages.

Reply:
I'm not sure, it's just a convention I used in a previous job. Amended.